scripts/create_tables.sql

```diff
@@ -1 +1,38 @@
 -- create doctors, patients, and appointments tables
+--doctors
+CREATE TABLE doctors (
+  id SERIAL PRIMARY KEY,
+  surname CHARACTER VARYING(128),
+  given_name CHARACTER VARYING(128),
+  gender CHARACTER VARYING(1),
+  dob DATE,
+  specialty CHARACTER VARYING(128),
+
+  doctor_id INTEGER REFERENCES doctors(id),
```

Comment:
Why would the `doctors` table need a second column to refer to another `id` value on the `doctors` table?